pkg/credentials/util.go

```diff
@@ -1,18 +1,43 @@
 package credentials
 
 import (
+	"fmt"
 	"path/filepath"
+
+	"github.com/gptscript-ai/gptscript/pkg/config"
+	runtimeEnv "github.com/gptscript-ai/gptscript/pkg/env"
 )
 
 type CredentialHelperDirs struct {
 	RevisionFile, LastCheckedFile, BinDir string
 }
 
-func GetCredentialHelperDirs(cacheDir string) CredentialHelperDirs {
+func RepoNameForCredentialStore(store string) string {
+	switch store {
+	case config.SqliteCredHelper:
+		return "gptscript-credential-sqlite"
```

Comment:
Nit: `gptscript-credential-sqlite` and `gptscript-credential-helpers` shows up multiple and might be a good idea to define those as const.

Reply:
I might change this at a later point. Don't want to go through the process of getting approvals on this one again just for that change